conf/vertebrates/mlss_conf.xml

```diff
@@ -98,34 +103,206 @@
   <multiple_alignments>
 
     <!-- Primates -->
-    <multiple_alignment method="EPO+EPO_EXTENDED">
+    <multiple_alignment method="EPO">
+      <species_set name="primates">
+        <genome id="60"/>
```

Comment:
I cannot check all of these `genome_db_id` so just assuming you have them all correct. If a grant asks for another genome to be included, it just gets stuck in here/swapped out with another one?

Reply:
If we as with plants, it should be swapped. But we will see.